__tests__/user.test.ts

```diff
@@ -57,7 +56,7 @@ const mockUser = {
 	isBlindMode: true,
 	doesNotPreferHelp: false,
 	readsBraille: true,
-	token: createToken(),
```

Comment:
OHHHH you did change it, this PR wasn't merged in yet though which is why I still saw `createToken()` in my version of this file 🙃